java/vector/src/main/java/org/apache/arrow/vector/BaseVariableWidthViewVector.java

```diff
@@ -46,7 +46,7 @@
  */
 public abstract class BaseVariableWidthViewVector extends BaseValueVector implements VariableWidthFieldVector {
   // A single element of a view comprises 16 bytes
-  protected static final int ELEMENT_SIZE = 16;
+  public static final int ELEMENT_SIZE = 16;
```

Comment:
@lidavidm I have doubts with this approach, should we use getters?

Reply:
a constant should not need a getter

Reply:
making `protected` to `public` okay?

Reply:
yup